testsupport/tiers/checks.go

```diff
@@ -216,7 +216,7 @@ func (a *base1nsTierChecks) GetNamespaceObjectChecks(_ string) []namespaceObject
 	checks := []namespaceObjectsCheck{
 		resourceQuotaComputeDeploy("20", "14Gi", "1750m", "14Gi"),
 		resourceQuotaComputeBuild("20", "10Gi", "2", "10Gi"),
-		resourceQuotaStorage("15Gi", "15Gi", "15Gi", "5"),
+		resourceQuotaStorage("50Gi", "50Gi", "50Gi", "12"),
```

Comment:
We are not changing (and we don't want to) the quota for other tiers. Only for the appstudio one.
```suggestion
		resourceQuotaStorage("15Gi", "15Gi", "15Gi", "5"),
```